packages/terra-application-utility/examples/index-examples/ApplicationHeaderUtilityExample.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Image from 'terra-image';
+import MockConfig from './MockConfig';
+import FallbackAvatar from './FallbackAvatar.svg';
+import { ApplicationHeaderUtility, UtilityUtils } from '../../lib/ApplicationUtility';
+
+class ApplicationHeaderUtilityExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onDiscloseUtility = this.onDiscloseUtility.bind(this);
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.state = {
+      discloseCount: 0,
+      selectedKey: null,
+    };
+  }
+
+  onDiscloseUtility() {
+    this.setState({ discloseCount: this.state.discloseCount += 1 });
+  }
+
+  handleOnChange(event, key) {
+    this.setState({ selectedKey: key });
+  }
+
+  render() {
+    const accessory = <Image src={FallbackAvatar} />;
+    const title = 'User Name';
+
+    return (
+      <div style={{ paddingLeft: '4px' }}>
+        <div style={{ height: '60px', position: 'relative', width: '150px', backgroundColor: 'bisque' }}>
```

Comment:
If not too much trouble, lets update this the `backgroundColor: 'dodgerblue'`, otherwise is fine to merge this PR as-is and we can add in a future (much smaller) PR, either way is fine.